_posts/2015-02-24-how-to-use-github-and-the-terminal-a-guide.md

```diff
@@ -0,0 +1,283 @@
+
+>**Congratulations! A lot of the steps that you've just done are one time steps. You only have to install Homebrew >once. You only have to make SSH keys once. You only have to clone the repo once. Just wanted to keep your morale >high. Onward!**
+
+### Create A Branch 
+
+Okay. Now you're ready to start editing. 
+
+*Open up Sublime Text or any text editor. 
+
+*Now open your copy of 18f.gsa.gov, which is located in the directory `user/code` where user is your username.
+
+*Click on 18f.gsa.gov and click open. 
+
+>Voila! You can now see all of the files that make up the site. It should look like this.
+
+![Screen Shot: All of the files that make up the 18F site](https://cloud.githubusercontent.com/assets/5784474/6352212/0699ef96-bc0f-11e4-95b8-07338522b71c.png)
+
+>We now want to create a branch where you can make changes to the website. A branch is basically a way to make >changes to a file that only belong to you. Someone else has to approve those changes before they go live on the >site. This allows you to collaborate with teammates without intefering with other people's existing work. 
+
+*To create a new branch, type `git checkout -b your-name-post` (The name of the branch can be anything but it should be descriptive.) 
+
+>This also moves you onto that branch.
+
+*Type `git status` to make sure you're now on the branch. You should see something that looks like this:
+
+![Screenshot: git checkout and status](https://cloud.githubusercontent.com/assets/5784474/6352442/b6f0e16e-bc10-11e4-830a-c9b62750f483.gif)
+
+### Edit and Post a Blog Post
+
+We're now going to walk you through creating a new blog post for 18f.gsa.gov. 
+
+*Type `ls -l` to see a list of files and folders that make up 18f.gsa.gov's repo. 
+
+>On the far left, you'll see `-rw-r--r--`. Any line that starts with a d is a directory. 
+
+
+![screen shot of 18F directories](https://cloud.githubusercontent.com/assets/5784474/6352518/50506686-bc11-11e4-8adc-b021c047584f.png)
+
+>The far right column is the name of the file or directory. 
+
+*You can also type in the command `ls -F` which shows you all of the files and is a lot less noisy. In this case, anything that ends with a `/` is a directory:
+
+![screen shot: ls -F command](https://cloud.githubusercontent.com/assets/5784474/6352713/846d2a8e-bc12-11e4-81f7-5cb433f1106d.png)
+
+
+*Go back to Sublime Text and write your post. Then save your post in the `_posts` directory with the name `2015-02-19-new_post.md`
+
+![Screen shot: Saving post in _posts directory](https://cloud.githubusercontent.com/assets/5784474/6355905/a07c1df6-bc27-11e4-84a1-ae08b0c84cd0.png)
+
+
+### Adding Front Matter
+
+The next step is to add what's called front matter. This is metadata for the blog post, and includes things like the title, authors, description, and date. You add front matter by typing in :
+
+```yaml
+--- on line 1.
+Title:New Post on line 2.
+Layout:Post on line 3.
+--- on line 4.
```

Comment:
the keys, the part before the colon, should be lower case and there should be a space between the `:` and the value (part to the right). We're also missing some metadata that posts normally use.
```yaml
---
title: New post
layout: post
tags:
- Tag
- Something else
- A third topic
excerpt: ""
description: ""
image: /path/to/image.png
---
```